### DISCUSSION TITLE
Docker engine EOL mechanism

### DISCUSSION THREAD

**Discussion by ray20013247777** [moby/moby#48431] [Q&A] [answered]
Hello:

I would like to ask if there is a clear end-of-life (EOL) mechanism for Docker Engine at the moment, as I urgently need information on this topic. My current source of information is https://endoflife.date/docker-engine, but the website clearly states:
```
Docker Engine is supported by the Moby Community, and as such - there’s no list of supported releases.
This page uses the Open milestones as a guide for which releases are still getting fixes.
```
We are currently using version 24.0.x, and I would like to confirm whether the closing of a milestone is equivalent to declaring EOL, or if it just means there are no further feature plans for that branch. If there are related vulnerability issues, will they still be addressed with fixes?

Thank you!

**Comment by vvoland** [comment, marked as answer]
Docker Inc. only maintains the last major release branch (`27.x` currently).

See: https://github.com/moby/moby/blob/master/project/BRANCHES-AND-TAGS.md

Other unmaintained branches also sometimes receive security fixes (like the recent CVE-2024-41110), but no new packages are built and no releases are tagged.
